### PR TITLE
Support testing in VKS environment.

### PR DESCRIPTION
**Pull request by blackpiglet**

### CODE REVIEW

File: kubernetes/yaml/vsphere/kibishiiCNSStorageClass.yaml [hunk 1]
```diff
@@ -5,8 +5,9 @@ metadata:
   annotations:
     storageclass.kubernetes.io/is-default-class: "false"
 parameters:
-  StoragePolicyName: "vSAN Default Storage Policy"
-  # DatastoreURL: "ds:///vmfs/volumes/vsan:52cdfa80721ff516-ea1e993113acfc77/" # Optional Parameter
+  # StoragePolicyName: "vSAN Default Storage Policy"
+  # DatastoreURL: "ds:///vmfs/volumes/v456456san:52cdfa80721ff516-ea1e993113acfc77/" # Optional Parameter
```
Comment (reasonerjt) [inline kubernetes/yaml/vsphere/kibishiiCNSStorageClass.yaml]: is this "456456" a typo?

**Review by reasonerjt — APPROVED** [review]

**Comment by renmaosheng** [comment]
I think we just want to make the VKS as default for testing? tkgm will be tested separately by tkgm team, but not deprecating, right?

File: kubernetes/yaml/vsphere/kibishiiCNSStorageClass.yaml (hunk 1 above)
Comment (blackpiglet) [reply kubernetes/yaml/vsphere/kibishiiCNSStorageClass.yaml]: Thanks. Modified.

**Comment by blackpiglet** [comment]
Yes. The wording is not accurate enough.
I mean we stop supporting TKGm environment testing in the Kibishii repository.

**Comment by reasonerjt** [comment]
@blackpiglet Maybe it's better to write some instructions in README.md to help the tester understand what need to be updated to have kibushii run on TKGm?

**Comment by blackpiglet** [comment]
@reasonerjt @renmaosheng 
Updated. PTAL.

**Pull request merged**